06_control_statements_looping/exercises/array.c

```diff
@@ -0,0 +1,45 @@
+// Write a program that creates two eight-element arrays of doubles and uses 
+// a loop to let the user enter values for the eight elements of the first 
+// array. Have the program set the elements of the second array to the 
+// cumulative totals of the elements of the first array. For example, the 
+// fourth element of the second array should equal the sum of the first four 
+// elements of the first array, and the fifth element of the second array 
+// should equal the sum of the first five elements of the first array. (It’s 
+// possible to do this with nested loops, but by using the fact that the fifth 
+// element of the second array equals the fourth element of the second array 
+// plus the fifth element of the first array, you can avoid nesting and just 
+// use a single loop for this task.) Finally, use loops to display the 
+// contents of the two arrays, with the first array displayed on one line and 
+// with each element of the second array displayed below the corresponding 
+// element of the first array.
+#include <stdio.h>
+
+#define SIZE 8
+
+int main(void)
+{
+    double first[SIZE];
+    double second[SIZE];
+    int index;
+    
+    for (index = 0; index < SIZE; index++)
+    {
+        printf("Please enter element of the first array: ");
+        scanf("%lf", &first[index]);
+    }
+   
+    for (index = 0; index < SIZE; index++)
+    {
+        printf("%.2lf ", first[index]);
+    }
+    printf("\n%.2lf ", first[0]);
```

Comment:
Here you print the first element of the first array instead of the first element of the second array. They must be equal, but if the program contains a mistake and these two elements aren't equal we will never know it. Please display the second array in a separate loop.

And this is not because I am a mushroom, but because this is the right way to do it=)